### PR TITLE
chore(FDS-446):[Docs]: Fix invalid labels in the Documentation

### PR DESCRIPTION
**Pull request by rociobaena**
Fix invalid labels in the Documentation and include AreaPlaceholder to the mdx_components.
## PR Author Checklist

- [x] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
- [x] I have finished reviewing the contents of this PR for accuracy
- [x] If needed, I have included a [Changeset](https://github.com/changesets/changesets) and it follows [Semantic Versioning principles](https://semver.org/)
- [x] I have completed all of the above and have enabled `auto-merge` for this PR

### CODE REVIEW

File: packages/cascara/src/modules/DataTextArea/DataTextArea.doc.mdx [hunk 1]
```diff
@@ -22,9 +22,9 @@ Feel free to play around with the following code:
   }}
 >
   <DataTextArea
-    attribute='firstName'
+    attribute='Textarea'
     isEditable={true}
-    label='First Name'
+    label='Textarea'
```
Comment (brianespinosa) [inline packages/cascara/src/modules/DataTextArea/DataTextArea.doc.mdx:27]:
```suggestion
    label='TextArea'
```
I think we should consider trying to keep some consistency with these labels and the components themselves. This probably means we should remove "Data" from the component name and use that as the label.

**Review by brianespinosa — CHANGES_REQUESTED** [review]
Small request for changes with the naming just so we have a 100% consistent pattern.

File: packages/cascara/src/modules/DataTextArea/DataTextArea.doc.mdx (hunk 1 above)
Comment (rociobaena) [reply packages/cascara/src/modules/DataTextArea/DataTextArea.doc.mdx:27]: We have Data in all our module input components.

File: packages/cascara/src/modules/DataTel/DataTel.doc.mdx [hunk 2]
```diff
@@ -23,9 +23,9 @@ Feel free to play around with the following code:
   }}
 >
   <DataTel
-    attribute='state'
+    attribute='telephone'
     isEditable={true}
-    label='State'
+    label='Telephone'
```
[the remark this answers is not in the document]
Comment (rociobaena) [reply packages/cascara/src/modules/DataTel/DataTel.doc.mdx]: Done

File: packages/cascara/src/modules/DataTel/DataTel.doc.mdx [hunk 3]
```diff
@@ -23,9 +23,9 @@ Feel free to play around with the following code:
   }}
 >
   <DataTel
-    attribute='state'
+    attribute='telephone'
```
[the remark this answers is not in the document]
Comment (rociobaena) [reply packages/cascara/src/modules/DataTel/DataTel.doc.mdx]: Done

File: packages/cascara/src/modules/DataTextArea/DataTextArea.doc.mdx (hunk 1 above)
Comment (brianespinosa) [reply packages/cascara/src/modules/DataTextArea/DataTextArea.doc.mdx]: I am talking specifically about the component name "DataTextArea" removing "Data" so the label prop is just "TextArea" with the same capitalization.

Comment (rociobaena) [reply packages/cascara/src/modules/DataTextArea/DataTextArea.doc.mdx]: Done

**Review by brianespinosa — APPROVED** [review]
LGTM!

**Review by lordpixel — APPROVED** [review]

**Pull request merged**